GameDesign/Assets/Scripts/MonoBehaviours/LevelMonobehaviour.cs

```diff
@@ -69,9 +80,9 @@ void Update()
         }
     }
 
-    void OnTimeFinished()
-    {
-        Debug.Log("Time's over! Level finished");
-        Time.timeScale = 0;
+    void OnTimeFinished()
+    {
+        Debug.Log("Time's over! Level finished");
```

Comment:
maybe triggering an event would be better here. The event could start an animation or simply show the UI. but ofc its not necessary to change it yet.

Reply:
This is already subscribed to an event. The event is being triggered by the timer manager. We need how to integrate the other losing condition